warn/bzlenv.go

```diff
@@ -0,0 +1,218 @@
+// Package bzlenv provides function to create and update a static environment.
+package bzlenv
```

Comment:
FYI: adding a second package to the `warn/` directory prevents the basic Go toolchain from building buildifier.

Reply:
Thanks for pointing out, I'll move it into the `warn` package.